docs/templates.md

```diff
@@ -0,0 +1,24 @@
+# Distributed Application Templates
+
+Distributed Applications (dApps) are basically decentralized applications. The templates can be found [here](https://github.com/scale-it/algo-builder-templates) and are extremely helpful and easy to use. They are designed and implemented in such a way that they can provide the aspiring developers a headstart in building dApps based on `Algorand Blockchain`. The templates heavily use [algosdk](https://github.com/algorand/js-algorand-sdk), [algo-builder](https://github.com/scale-it/algo-builder) and [AlgoSigner](https://github.com/PureStake/algosigner).
```

Comment:
I think we can remove first line.  Distributed..........applications. this line

Reply:
okay. will remove it.